### DISCUSSION TITLE
Google Play App Access and Permissions Declaration form

### DISCUSSION THREAD

**Discussion by andyrohan-fiserv** [fastlane/fastlane#17535] [Q&A] [answered]
Does Fastlane provide the ability to add credentials in the App Content -> App Access and App Content -> Sensitive Permissions Declaration form sections on Google Play Console as per the requirements here: https://support.google.com/googleplay/android-developer/answer/9799150?hl=en ?
I suspect the answer is "no" as I couldn't find any Google API to edit these sections but curious if anyone has looked into this?

**Comment by max-ott** [comment, marked as answer]
We're using the Google Play Publishing API and are therefore limited to its features: https://developers.google.com/android-publisher

As everything else would require a totally different approach (web scrapping and similar) this will not get added in the future soon.

**Reply by andyrohan-fiserv** [reply]
I suspected as much, thank you.
